proposals/function-pointers.md

```diff
@@ -176,6 +176,7 @@ In an unsafe context, an implicit conversion exists from an address-of expressio
 - A single method `M` is selected corresponding to a method invocation of the form `E(A)` with the following modifications:
    - The arguments list `A` is a list of expressions, each classified as a variable and with the type and modifier (`ref`, `out`, or `in`) of the corresponding _formal\_parameter\_list_ of `D`.
    - The candidate methods are only those methods that are only those methods that are applicable in their normal form, not those applicable in their expanded form.
+   - The candidate methods are only those methods that are static.
```

Comment:
It looks like the previous bullet should be adjusted: "those methods that are only those methods that are applicable"